src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -56,7 +56,7 @@ export const styles = StyleSheet.create({
   },
   btnAdd: {
     padding: dimen.normalizeDimen(8),
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the button has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could impact visibility for users with visual impairments.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->